remote-query/remote-query-server/src/test/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptorTest.java

```diff
@@ -318,7 +319,22 @@ public void testStateTransfer() {
 
    private void assertNoTransactionsAndLocks() {
       assertNoTransactions();
-      TestingUtil.assertNoLocks(cache(0));
-      TestingUtil.assertNoLocks(cache(1));
+      waitForNoLocks(cache(0));
+      waitForNoLocks(cache(1));
+   }
+
+   public void waitForNoLocks(Cache<?, ?> cache) {
```

Comment:
Should be `private`, TestNG thinks it's a test method
https://ci.infinispan.org/job/Infinispan/job/PR-8494/13/testReport/junit/org.infinispan.query.remote.impl/ProtobufMetadataManagerInterceptorTest/waitForNoLocks/